packages/wallet/src/test-utils/launchNode.ts

```diff
@@ -23,18 +24,22 @@ type LaunchNodeOptions = {
  * @param consensusKey - the consensus key to use.
  * @param dbPath - the path to the folder to use for the fuel-core db.
  * @param args - additional arguments to pass to fuel-core
+ * @param useSystemFuelCore - whether to use the system fuel-core binary or the one provided by the \@fuel-ts/fuel-core package.
  * */
 export const launchNode = async ({
   chainConfigPath,
   consensusKey,
   dbPath,
   args = defaultFuelCoreArgs,
+  useSystemFuelCore = true,
```

Comment:
I think the default should be `false` for increased compatibility <sup>(*)</sup>.

Only in atypical situations will users want otherwise.

> <sup>(*)</sup> _Same approach being applied in the Tooling_.

Reply:
Makes sense, mb!